### PR TITLE
Correcting translations

### PR DESCRIPTION
**Pull request by bobbrow**
Also removing the source of confusion for future translations

### CODE REVIEW

**Comment by Colengms** [comment]
Just a thought: If you have a substring that doesn't need to be localized, you can break it out of the loc string and use a placeholder (`{0}`, `{1}`) in it's place.  Translators should be familiar with that style of placeholder.

**Comment by bobbrow** [comment]
I think that's what I did in this PR (in addition to updating the existing translations). You have to scroll down to the last file. 😄

**Review by Colengms — APPROVED** [review]

File: i18n/plk/src/preset.i18n.json [hunk 1]
```diff
@@ -9,8 +9,8 @@
 	"default.test.preset": "[Domyślna]",
 	"default.test.preset.description": "Pusty predefiniowany test, które nie dodaje żadnych argumentów",
 	"no.cl.arch": "Predefiniowana konfiguracja {0}: nie określono architektury dla elementu cl.exe, używając domyślnej opcji x86",
-	"no.cl.toolset.arch": "Skonfiguruj ustawienie wstępne {0}: nie określono zestawu narzędzi dla elementu cl.exe, używając domyślnej opcji x86",
-	"invalid.cl.toolset.arch": "Skonfiguruj ustawienie wstępne {0}: Określono nieoczekiwaną architekturę zestawu narzędzi \"{1}\" czy chodziło Ci o \"host ={1}\"?",
+	"no.cl.toolset.arch": "Skonfiguruj ustawienie wstępne {0}: nie określono zestawu narzędzi dla elementu cl.exe, używając domyślnej opcji host=x86",
```
Comment (lygstate) [inline i18n/plk/src/preset.i18n.json:12]: weird, this is not updated

Comment (bobbrow) [reply i18n/plk/src/preset.i18n.json:12]: I added "host=" to that line, but it's not highlighted for some reason.

**Pull request merged**